modules/KalturaSupport/resources/mw.ClosedCaptions.js

```diff
@@ -577,8 +577,8 @@
 				captionsSrc = mw.getConfig('Kaltura.ServiceUrl') +
 							"/api_v3/index.php/service/caption_captionasset/action/serveWebVTT/captionAssetId/" +
 							dbTextSource.id +
-							"/segmentIndex/-1/version/2/captions.vtt";
-				captionsSrc += ks ? '/ks/' + ks : '';
+							"/segmentIndex/-1/version/2";
+				captionsSrc += ks ? "/ks/" + ks + "/captions.vtt" : "/captions.vtt";
```

Comment:
```suggestion
				captionsSrc += ks ? "/ks/" + ks : "" + "/captions.vtt";
```

Reply:
@yairans 
We need the .vtt extension (/captions.vtt) at the end in order to recognize it as a VTT file.

Will work:

`http://cdnapi.kaltura.com/api_v3/index.php/service/caption_captionasset/action/serveWebVTT/captionAssetId/0_svo0de7e/segmentIndex/-1/version/2/ks/NjdlOTkwMWFkYzNlNTNiMjgyZTczMTBjNWE1MGQ2OGU2OTcyMDExYnw1NDIxMzE7NTQyMTMxOsadfdsdfsdfzY7MjsxNTc5NDQ0Njc2LjY5NDY7YWRtaW47ZGlzYWJsZWVudGl0bGVtZW50Ozs=/captions.vtt`

Will not work:
http://cdnapi.kaltura.com/api_v3/index.php/service/caption_captionasset/action/serveWebVTT/captionAssetId/0_svo0de7e/segmentIndex/-1/version/2/ks/NjdlOTkwMWFkYzNlNTNiMjgyZTczMTBjNWE1MGQ2OGU2OTcyMDExYnw1NDIxMzE7NTQyMTMxOsadfdsdfsdfzY7MjsxNTc5NDQ0Njc2LjY5NDY7YWRtaW47ZGlzYWJsZWVudGl0bGVtZW50Ozs=